tests/regression/test_regression.py

```diff
@@ -108,6 +108,22 @@ class TestMultiClassCls(BaseTest):
         )
         for idx in range(1, 4)
     ]
+    DATASET_TEST_CASES.extend([
```

Comment:
Can we abbreviate it such as

```python
DATASET_TEST_CASES = [  # noqa: RUF012
        DatasetTestCase(
            name=f"multiclass_CUB_small_{idx}",
            data_root=Path("multiclass_CUB_small") / f"{idx}",
            data_format="imagenet_with_subset_dirs",
            num_classes=2,
            extra_overrides={"trainer.max_epochs": "20"},
        )
        for idx in range(1, 4)
    ] + [
        ...
    ]
```

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/2770/commits/532fe304bdfe0759b5c34befd10a0be8c0daa10f. BTW, just curiosity: is it better to read?

Reply:
The difference between the previous one and this one is that

1) The previous one makes `DATASET_TEST_CASES` look like a **mutable variable**. At the runtime, it is expanded by the next line `.expand()`.
2) For this one, there is no change of `DATASET_TEST_CASES` after its creation, so that it looks like an **immutable variable**.

It is important to distinguish between mutable and immutable variables.

Reply:
Thanks for the comment :)